articles/sso/current/index.md

```diff
@@ -8,7 +8,7 @@ toc: false
 Single Sign On (SSO) occurs when a user logs in to one application and is then signed in to other applications automatically, regardless of the platform, technology, or domain the user is using.
 
 ::: note
-The [Hosted Login Page](/hosted-pages/login) is the easiest and most secure way to implement SSO with Auth0. If you need to use embedded [Lock](/libraries/lock) or an embedded custom authentication UI in your application, check out the [Cross-Origin Authentication](/cross-origin-authentication) page for information on safely conducting SSO.
+[Universal login](/hosted-pages/login) is the easiest and most secure way to implement SSO with Auth0. If you need to use embedded [Lock](/libraries/lock) or an embedded custom authentication UI in your application, check out the [Cross-Origin Authentication](/cross-origin-authentication) page for information on safely conducting SSO.
 :::
```

Comment:
This is not accurate, the COA page does not help much in safely conduct SSO. However, there have been a lot of conversations regarding SSO in Embedded scenarios, we need to still iterate on finding the best way to communicate it.